### PR TITLE
YJ_006: Setup Gov.uk styling and template site

### PR DESCRIPTION
**Pull request by MatthewJKainos**

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -5,19 +5,34 @@ import session from "express-session";
...
 
 app.use(bodyParser.json())
 app.use(bodyParser.urlencoded({
   extended: true
 }))
 
-app.use(session({ secret: 'SUPER_SECRET', cookie: { maxAge: 28800000 }}));
+app.use(session({ secret: 'SUPER_SECRET', cookie: { maxAge: 28800000 } }));
+
+console.log(path.join(__dirname, '/node_modules/govuk-frontend/dist/govuk/assets'));
```
Comment (lukemKainos) [inline src/app.ts]: Console.log?

File: src/app.ts [hunk 2]
```diff
@@ -5,19 +5,34 @@ import session from "express-session";
...
+  express: app
 });
 
 app.use(bodyParser.json())
 app.use(bodyParser.urlencoded({
   extended: true
 }))
 
-app.use(session({ secret: 'SUPER_SECRET', cookie: { maxAge: 28800000 }}));
+app.use(session({ secret: 'SUPER_SECRET', cookie: { maxAge: 28800000 } }));
```
Comment (lukemKainos) [inline src/app.ts]: lets env var the secret rather than a hardcode

File: src/app.ts [hunk 3]
```diff
@@ -5,19 +5,34 @@ import session from "express-session";
...
+const path = require('path');
+
 const app = express();
 
-nunjucks.configure('views', {
-    autoescape: true,
-    express: app
+nunjucks.configure([
+  "node_modules/govuk-frontend/dist",
+  "views"
```
Comment (lukemKainos) [inline src/app.ts]: why double quoted?

**Review by lukemKainos — CHANGES_REQUESTED** [review]

File: src/app.ts (hunk 3 above)
Comment (MatthewJKainos) [reply src/app.ts]: Good question, why singled quoted?

Comment (lukemKainos) [reply src/app.ts]: because everything else in the file is single quoted

**Review by lukemKainos — APPROVED** [review]

**Review by Aaron-Millen — APPROVED** [review]

**Pull request merged**